### ISSUE TITLE
Entries not posting on Blogger

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#811]
```
What's the problem?
I typed up a very long blog entry recently, and clicked "Publish Post" but the 
entry did not post on my Blogger blog.




What browser are you using?
Firefox. Then I tried on Chrome. Same issue.




What version of ScribeFire are you running?
ScribeFire 4.2.3
```

-----
Original issue reported on code.google.com by `callmero...@gmail.com` on 27 Jan 2014 at 12:06

**Comment by GoogleCodeExporter** [comment]
```
I have been having the same problem.  Both on Chrome and Firefox.  Was working 
fine a couple weeks ago.  
```

-----
Original comment by `audreype...@gmail.com` on 30 Jan 2014 at 6:58

**Comment by GoogleCodeExporter** [comment]
```
Fixed for blogger blogspot in version 4.3.1
```

-----
Original comment by `scribefi...@gmail.com` on 14 Apr 2014 at 6:58
* Changed state: **Fixed**
